internal/helpers/protobuf.go

```diff
@@ -0,0 +1,101 @@
+package helpers
+
+import (
+	"io/ioutil"
+	"path/filepath"
+
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/types/descriptorpb"
+
+	"github.com/deviceinsight/kafkactl/output"
+	"github.com/jhump/protoreflect/desc"
+	"github.com/jhump/protoreflect/desc/protoparse"
+)
+
+type ProtoSearch struct {
+	ProtosetFiles    []string
+	ProtosetPaths    []string
+	ProtoFiles       []string
+	ProtoImportPaths []string
```

Comment:
Do you see a way to reduce the parameters here? This would then also simplify the configuration and commandline parameters.
I'm thinking of perhaps just two parameters `ProtoFiles`, `ProtoPaths` and for example to find protoset files just search for *.protoset in `ProtoPaths`.

Reply:
Merging `ProtosetFiles` with `ProtoFiles` will require more complex handling (need to determine file type). So it looks like 3 parameters: `ProtosetFiles`, `ProtoImportPaths` and `ProtoFiles` is reasonable minimum. Importing protoset files from `ProtoImportPaths` will be confusing for users imho.

Reply:
ok then I would say, we leave it like it is